src/components/scatterplot/Scatterplot.js

```diff
@@ -199,6 +277,14 @@ class Scatterplot extends AbstractSpatialOrScatterplot {
       this.onUpdateCellsLayer();
       this.forceUpdate();
     }
+    if ([
+      'cellSetPolygons', 'viewState', 'cellSetPolygonsVisible',
+      'cellSetLabelsVisible', 'cellSetLabelSize',
+    ].some(shallowDiff)) {
```

Comment:
Is `viewState` needed here?  The performance on FF seems much better without it and it seems ot be functional without it.

Reply:
Yes the viewState is needed, otherwise the collision detection does not work, for example if you zoom out 
![Screen Shot 2020-12-17 at 6 32 38 PM](https://user-images.githubusercontent.com/7525285/102557642-ac4f8a80-4099-11eb-9f9a-cf09a1b10d9e.png)

vs.

![Screen Shot 2020-12-17 at 6 32 51 PM](https://user-images.githubusercontent.com/7525285/102557652-b07ba800-4099-11eb-95b9-83ce96223392.png)